src/odemis/driver/test/emitter_test.py

```diff
@@ -63,11 +65,22 @@ def setUp(self):
         self.child2 = CHILD2_CLASS(**CHILD2_KWARGS)
         self.dev = emitter.MultiplexLight("test", "light",
                                           children={"c1": self.child1, "c2": self.child2})
+        
```

Comment:
Don't mix the tests for the ExtendedLight with MultiplexLight. With unittest, you can just have multiple classes in the module. Each of them  will be executed.
=> Have a separate `class TestExtendedLight(unittest.TestCase)`.